.gitignore

```diff
@@ -76,3 +76,6 @@ bin/fetch_config.rb
 
 # Visual Studio Code
 .vscode
+
+# Random test outputs
+ministerial-report-applications*.txt
```

Comment:
![](https://media.giphy.com/media/VfQhdUQpT19UthsTIf/giphy.gif)

Reply:
yw

Though we should probably change this to output into a tmp directory or something.